app/Annotations/OpenApi/controllersAnnotations/studentAnnotations/AnnotationsStudentDetail.php

```diff
@@ -5,66 +5,60 @@
 class AnnotationsStudentDetail
```

Comment:
The structure does not correspond to the output that was there before.
<img width="529" alt="OutputAnterior" src="https://github.com/IT-Academy-BCN/ita-profiles-backend/assets/106141089/8046cb89-f3c8-415d-b37c-cb572502db7b">

Reply:
I have modified the code and now it returns exactly what you ask for. Thank you for the review.